Makefile.toml

```
@@ -259,11 +260,19 @@ chmod -R o+r ${CARGO_HOME}
 dependencies = ["fetch-sdk"]
 script = [
 '''
-${BUILDSYS_TOOLS_DIR}/docker-go \
-  --module-path ${BUILDSYS_SOURCES_DIR}/host-ctr \
-  --sdk-image ${BUILDSYS_SDK_IMAGE} \
-  --go-mod-cache ${GO_MOD_CACHE} \
-  --command "go list -mod=readonly ./... >/dev/null && go mod vendor"
+function go_fetch
+{
+  local module=$1
```

I prefer this form for expected variables: put the `local` declaration on its own line, so it doesn't mask errors, and then do the assignment and error if no argument was passed in that position.

```suggestion
  local module
  module="${1:?}"
```